backend/api/models.py

```diff
@@ -52,7 +52,8 @@ def get_populations(self):
         populations = (
             Collection.objects.filter(dataset=self)
                               .distinct('population')
-                              .values_list('population', flat=True))
+                              .values_list('population', flat=True)
+                              .order_by('population'))
```

Comment:
Make this same change in `tests/test_models.py` line 25.

Reply:
Yep great catch.